.gitlab-ci.yml

```diff
@@ -311,9 +311,7 @@ OSTree simplified installer:
   parallel:
     matrix:
       - RUNNER:
-          - rhos-01/rhel-8.6-nightly-x86_64-large
```

Comment:
I deleted entries here to save CI runs. Will revert it once the PR is passing the oci integ test